src/json-schema/partial/common.json

```diff
@@ -158,6 +158,11 @@
                         "type": "number",
                         "description": "A number (>= 1) which is the multiplier that increases the retry interval on each attempt. (Default: 2.0)",
                         "minimum": 1.0
+                    },
+                    "Comment": {
```

Comment:
Is it possible to use a ref to "#/definitions/Comment" instead of redefining it in each block?

Reply:
done